python/cuml/fil/fil.pyx

```diff
@@ -183,37 +274,49 @@ class FIL(Base):
         threshold : threshold is used to for classification
            if output == OUTPUT_CLASS, else it is ignored
         """
-        return self._impl.from_treelite(model, output_class, algo, threshold)
+        return self._impl.from_treelite_model(model, output_class, algo, threshold)
 
 
-def from_treelite_direct(model, algo=0, output_class=True,
-                         threshold=0.5, handle=None):
-    """
-    Creates a FIL model using the treelite model
-    passed to the function.
-    Note : from cuml import fil
-           forest = fil.from_treelite_direct(params)
-           # in order to predict
-           predicted_labels = forest.predict(X_test)
-
-    Note : do not reuse or overwrite the tl_model variable which is
-       used to load a saved model.
-       This will cause Segmentation errors.
-       here tl_model = = treelite.Model.load('xgb.modle', 'xgboost')
+def load_fil_from_treelite_file(filename, algo=0, output_class=True, threshold=0.50,
+                                handle=None):
```

Comment:
Could the file type be added as a named parameter (with `xgboost` as the default)?